modules/ui/sources/window.es

```diff
@@ -121,6 +121,9 @@ export default class {
 
     }.bind(this));
 
+    //mock default FF function
+    this.popup.enableOneOffSearches = function() {}
```

Comment:
Hey Drew! 
We also tried to implement (mock) that method in our popup implementation but the startup issue remained. Here is the line - https://github.com/cliqz-oss/browser-core/blob/test-pilot/specific/firefox/cliqz@cliqz.com/chrome/content/components.xml#L60

We are fine with this "fix" anyhow so we can keep it like this.